hydrocarbon_problem/agents/sac/load_and_plot.py

```diff
@@ -0,0 +1,9 @@
+import pickle
```

Comment:
@steef12111 if you run train.py with the current setting, and then run this file, it will plot the history that got saved.